.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "deps/concordium-rust-sdk"]
 	path = deps/concordium-rust-sdk
 	url = ../concordium-rust-sdk.git
+[submodule "deps/concordium-browser-wallet"]
+	path = deps/concordium-browser-wallet
+	url = ../concordium-browser-wallet.git
+	branch = web3Id-all-together
```

Comment:
There is a feature branch now that should be used. `feature/web3-id` 

That's where everything is at.